pkg/watch/health.go

```diff
@@ -57,8 +58,9 @@ type PodWatch struct {
 // runs a CheckHealth routine to monitor the health of each
 // service and kills routines for services that should no
 // longer be running.
-func MonitorPodHealth(config *preparer.PreparerConfig, logger *logging.Logger, shutdownCh chan struct{}) {
+func MonitorPodHealth(config *preparer.PreparerConfig, shutdownCh chan struct{}) {
```

Comment:
by that I mean... I think one possible alternative could have been to change `*logging.Logger` to `logging.Logger` and still pass the logger in from `p2-preparer/main.go` - is there reason we should favor one approach over the other